docs/configs/workers.md

```diff
@@ -98,6 +100,7 @@ Contains data related to your deployment, including, most importantly, deal id,
 | `definition`     | string  | **Yes**  | CID of uploaded to IPFS App Definition, which contains the data about everything that you are trying to deploy, including spells, service and module configs and CIDs for service wasms |
 | `timestamp`      | string  | **Yes**  | ISO timestamp of the time when the worker was deployed                                                                                                                                  |
 | `chainNetwork`   | string  | No       | DEPRECATED. Blockchain network name that was used when deploying workers Possible values are: `dar`, `kras`, `stage`, `local`.                                                          |
+| `matched`        | boolean | No       | Is deal matched                                                                                                                                                                         |
```

Comment:
@elshan-eth do you think this method without using indexer is a reliable way to tell if deal is matched or not?

Reply:
It also depends on your need.
1. If you need to check that deal has attached compute unit to it rn(it could be unattached later), then blockchain will do the work.
2. If you need to check whether deal has ever been matched and you don't care about status, then you need to use indexer.

Reply:
I don't care about current status I need to know if it ever was matched so I can either try to rematch or learn that there is no need to match

Reply:
Right now we need to store this information locally but we will add a public method on the contract in the next release

Reply:
I'm ok either way then